views_euid/index.hbs

```diff
@@ -237,6 +232,10 @@
           </button>
         </div>
       </form>
+      <div class="this-site--net-160765-C61RwL raleway-medium-stratos-12px">
+        {{__ "For more information on how the Trade Desk manages EUID and your rights with respect to EUID, please read the "}}
```

Comment:
From localisation perspective, in another language there may need to be some text after the link (and full stop should be part of localised string too). Is it possible to have parameters inside the string to inject the html tags into?

Reply:
The current framework doesn't have support for that ready to go - the current options are to put the markup into the translation files (which I advise we avoid), or do some extra work on finding the best option for markup within translations. I recommend we go with the latter, and defer it until we start adding additional languages.